docs/content/documentation/modules/tensor_mechanics/VolumetricLocking.md

```diff
@@ -0,0 +1,48 @@
+#Volumetric locking correction
+
+Volumetric locking is the over stiffening of elements when the material is close to being incompressible (Poisson's ratio $\nu$ nearing 0.5). This happens when a fully integrated element (such as Hex8 elements with 8 quadrature points or Quad4 elements with 4 quadrature points) is used. This is a numerical artifact introduced because shape functions used in finite element analysis cannot properly approximate the incompressibility condition throughout the element. To avoid this locking of elements, B-bar correction [see [Hughes(1987)](https://scholar.google.com/citations?view_op=view_citation&hl=en&user=6tPdlYAAAAAJ&citation_for_view=6tPdlYAAAAAJ:u5HHmVD_uO8C)] is implemented in MOOSE.
```

Comment:
Even better would be http://mooseframework.org/docs/PRs/8387/site/utilities/documentation/moose_flavored_markdown/index.html#bibliographies !